### PR TITLE
Develop to Main: small changes

### PR DESCRIPTION
**Pull request by YarinsHyper**
removed update method, made create method upsert.
new generated proto files.
const parameters starts with upper-case letters.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+COPY . .
+RUN make build
+
+#final stage
+FROM golang:alpine
+COPY --from=builder /go/src/app/birthday-service /birthday-service
+LABEL Name=birthday-service Version=0.0.1
+WORKDIR /go/src/app
+EXPOSE 8000
+ENTRYPOINT ["/birthday-service"]
```
Comment (yoavrozner) [inline Dockerfile]: # final stage
FROM scratch
COPY --from=builder /go/src/app/permission-service /permission-service
COPY --from=builder /bin/grpc_health_probe /bin/grpc_health_probe
LABEL Name=permission-service Version=0.0.1
EXPOSE 8080
ENTRYPOINT ["/birthday-service"]

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  mongo-express:
+    image: mongo-express
+    restart: always
+    ports:
+      - 8081:8081
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: root
+      ME_CONFIG_MONGODB_ADMINPASSWORD: example
+    depends_on:
+      - mongo
```
Comment (yoavrozner) [inline docker-compose.yml]: Redundant

**Review by yoavrozner — COMMENTED** [review]
Good job @yarinBenisty !
See my notes

File: docker-compose.yml (hunk 2 above)
Comment (YarinsHyper) [reply docker-compose.yml]: i used express as a ui for the mongo data, but i'll remove it.

File: Dockerfile (hunk 1 above)
Comment (YarinsHyper) [reply Dockerfile]: done

**Review by yoavrozner — APPROVED** [review]

**Review by Shahar-Y — APPROVED** [review]
Good job. Approved.

**Pull request merged**